app/templates/docs/vendor-sparql-api.hbs

```diff
@@ -62,7 +62,10 @@
   <strong>Noteer:</strong> voor de status is aangeraden om steeds met de URI te werken. Het label kan veranderen, maar het achterliggende concept (i.e. de URI) niet.
 </p>
 <p>
-  Met het volgende voorbeeld worden er 100 eerder verrrichtte meldingen opgevraagd. Met de verkregen URI's kan men queries gelijk de bovenstaande opstellen.
+  <strong>Noteer:</strong> de status voor een inzending kan "verzonden" of "concept" zijn. Een inzending in "concept" vereist nog manuele interventie door onvolledigheid van het document alvorens het officieel kan worden ingediend, waarna het de status "verzonden" krijgt. Verder op deze pagina worden de URI's voor de status opgelijst.
```

Comment:
Kan je nog iets zetten in de trant van . Indien er ingestuurd werd met status 'verzonden' en het de status blijft hangen op concept, is er hoogst waarschijnlijk iets mis met de ingestuurde data. U kan steeds terecht bij op
digitaalABB@vlaanderen.be  voor hulp